### DISCUSSION TITLE
How do I change the "Ollama" chatbot name?

### DISCUSSION THREAD

**Discussion by oliverbob** [open-webui/open-webui#106] [General]
Hi, I tried working with the ui. It works smoothly on localhost, but I'd like to customize it.

Whereas chatgpt has "icon" for this, I'd like to know where to find the directive to change the chatbot name. Or maybe I'm looking for the file that's responsible for the general settings that controls the display, icons, images, title, html, etc.

Any help would be appreciated.

Thanks.

**Comment by OneMonkeyFist** [comment]
Not sure about the other aspects - as I have not searched real hard yet. But the easiest way I found to change the name is to make a copy and then delete the original.

ollama cp original-llm new-name-llm
